src/ape/managers/chain.py

```diff
@@ -1196,6 +1202,44 @@ def instance_at(
             else:
                 raise  # Current exception
 
+        if abi:
+            # if abi is type string then convert it to json object
+            if isinstance(abi, str):
+                try:
+                    # convert the abi from string to list
```

Comment:
Some of these comments are distracting, let the code document itself and only use comments to show missing intent.

```suggestion
```